packages/back-end/src/services/stats.ts

```diff
@@ -65,20 +65,25 @@ export interface AnalysisSettingsForStatsEngine {
   phase_length_days: number;
   alpha: number;
   max_dimensions: number;
+  decision_metric: string;
+  bandit: boolean;
+  bandit_weights_seed: number;
```

Comment:
either move to BanditSettings... or move all BanditSettings to ANalysisSettings?

Reply:
moved to `BanditSettingsForStatsEngine`